src/components/modal/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import {
+    Modal as ChakraModal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    Text,
+    Spacer,
+} from "@chakra-ui/react";
+import { Button } from "..";
+
+export interface ModalProps {
+    isOpen: boolean;
+    onCancel: () => void;
+    onConfirm: () => void;
+    header: string;
```

Comment:
Consider making `header` to `header?` and give it a default value of `""` to make it optional in case the modal just needs to be a custom component.